src/hvm.c

```diff
@@ -302,9 +307,18 @@ u32 global_sum(u32 x) {
 
 // TODO: write a time64() function that returns the time as fast as possible as a u64
 static inline u64 time64() {
+
+// if not on windows
+#ifndef _WIN32
   struct timespec ts;
   clock_gettime(CLOCK_MONOTONIC, &ts);
   return (u64)ts.tv_sec * 1000000000ULL + (u64)ts.tv_nsec;
+#else
+  int64_t wintime;
+  GetSystemTimeAsFileTime((FILETIME*)&wintime);
```

Comment:
We should use `QueryPerformanceCounter` here. GetSystemTimeAsFileTime is non-monotonic

Reply:
If targeting at at least Windows 8, GetSystemTimeAsFileTime is much better I think.